src/vs/workbench/electron-main/menus.ts

```diff
@@ -392,6 +392,9 @@ export class VSCodeMenu {
 	}
 
 	private setOpenRecentMenu(openRecentMenu: Electron.Menu): void {
+		openRecentMenu.append(this.createMenuItem(nls.localize({ key: 'miReopenClosedFile', comment: ['&& denotes a mnemonic'] }, "&&Reopen Closed File"), 'workbench.files.action.reopenClosedFile'));
+		openRecentMenu.append(__separator__());
```

Comment:
@Tyriar careful, the menu might not have any entries (if there are not recent files) so you would have one separator too much in the end